tests/unit/amo/components/TestCategory.js

```diff
@@ -124,7 +124,12 @@ describe('Category.mapStateToProps()', () => {
   });
 
   it('passes the search state if the filters and state matches', () => {
-    store.dispatch(searchStart({ filters, results: [] }));
+    store.dispatch(searchStart({
```

Comment:
This should use a helper like `_searchStart()` that sets default argument values so that you aren't repeating yourself with arguments that are not relevant to what you are testing.